packages/data-point-express/lib/route-middleware.js

```diff
@@ -3,7 +3,7 @@ const url = require('url')
 const Middleware = require('./middleware')
 
 function dataPointEntityRoute (dataPoint, entityId, req, res, next) {
-  const pathname = url.parse(req.url).pathname
```

Comment:
Updating this was not straight forward, because of this I added the `eslint-disable-line` and created an issue to address individually https://github.com/ViacomInc/data-point/issues/352